client/src/helpers/test/ClassList.spec.js

```diff
@@ -0,0 +1,52 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import ClassList from '../../outside/components/ClassList.vue'
+import VueRouter from 'vue-router'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import {
+  faDownload,
+} from '@fortawesome/free-solid-svg-icons'
```

Comment:
Do we need the font here?  Please remove if it is not needed.

Reply:
We do. Since the proper component is used. If we don't then the test will throw an error regarding the unidentified component